Core/CoreTiming.cpp

```diff
@@ -314,7 +314,8 @@ void ScheduleEvent(s64 cyclesIntoFuture, int event_type, u64 userdata)
 	Event *ne = GetNewEvent();
 	ne->userdata = userdata;
 	ne->type = event_type;
-	ne->time = GetTicks() + cyclesIntoFuture;
+        ne->time = GetTicks();
+        ne->time += cyclesIntoFuture;
```

Comment:
I don't think this change is worth making. The old code is clear and doesn't actually have any problems.